modules/game_interface/widgets/uiitem.lua

```diff
@@ -29,9 +29,11 @@ function UIItem:onDrop(widget, mousePos)
 
     local itemPos = item:getPosition()
     local itemTile = item:getTile()
+	if not itemPos then modules.game_actionbar.onDragReassign(widget, item) return false end
     if itemPos.x ~= 65535 and not itemTile then return false end
 
     local toPos = self.position
+	if not toPos then modules.game_actionbar.onChooseItemByDrag(self, mousePos, item) return false end
```

Comment:
@scopz I don't like the way these 2 changes are handled, I think it should check if your dragging to an actionbar slot first, but I don't know how. If you do, I think it'd be beneficial.